adb_pywrapper/adb_device.py

```diff
@@ -1,6 +1,7 @@
 import subprocess
 from os import makedirs
 from os.path import basename, isfile
+from shlex import quote
```

Comment:
Just checking: `shlex` is part of the Python stdlib?

Reply:
It is :)